3MI0800270/2023-03-07/zad5.cpp

```diff
@@ -0,0 +1,335 @@
+#include <iostream>
+#include <fstream>
+#include <cstring>
+using namespace std;
+
+const char FILE_PATH[10] = "zad5.txt";
+
+struct User {
+    char username[33];
+    char password[33];
+    char email[33];
+};
+
+int getTableSize(const char* filepath) {
+    ifstream file(filepath);
+    int count = 0;
+    while (!file.eof()) {
+        char ch = file.get();
+        if (ch == '\n') {
+            count++;
+        }
+    }
+    file.close();
+    return count;
+}
+
+void readUser(User& user, ifstream& file) {
+    char str[100];
+    char* ptr = str;
+
+    if (!file.eof()) {
+        file.getline(str, 99, '\n');
+    }
+    char username[33];
```

Comment:
What's with those pointer arithmetic etc. Is it really that bad to use a simple `for loop` ?